frontend/src/app/pages/login-page/login-page.component.html

```diff
@@ -1,87 +1,106 @@
-<div class="login-side">
-    <mat-card *ngIf="loginPageMode == LoginPageMode.LOG_IN">
-        <mat-card-header>
-            <mat-card-title>Sign in</mat-card-title>
-        </mat-card-header>
-        <mat-card-content>
+<!--suppress XmlDuplicatedId -->
+<div class="row">
+    <header class="col-sm-12 col-lg-4">
+        <div class="logo">
+            <img src="../../../assets/icon/medtagger_logo.svg"/>
+            <h1>MedTagger</h1>
+        </div>
+        <h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam at leo vitae est
```

Comment:
Damn I really love Lorem ipsum but we need to change it to some MedTagger related text :)